stdlib/_ctypes.pyi

```diff
@@ -195,24 +195,44 @@ class CFuncPtr(_PointerLike, _CData, metaclass=_PyCFuncPtrType):
 _GetT = TypeVar("_GetT")
 _SetT = TypeVar("_SetT")
 
-# This class is not exposed. It calls itself _ctypes.CField.
-@final
-@type_check_only
-class _CField(Generic[_CT, _GetT, _SetT]):
-    offset: int
-    size: int
-    if sys.version_info >= (3, 10):
+if sys.version_info >= (3, 14):
+    @final
+    @type_check_only
+    class _CField(Generic[_CT, _GetT, _SetT]):
```

Comment:
Shouldn't we be able to name this `CField`? After all, this exists on Python 3.14+:

```
Python 3.14.0 (main, Oct  8 2025, 10:57:17) [GCC 13.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from _ctypes import CField
```

Then we'd need to tweak the imports in `ctypes` a bit, but this is closer to the implementation for 3.14+.

Reply:
Renaming that to `CField` is going to break some more code, 
https://github.com/search?q=repo%3Apython%2Ftypeshed%20_CField&type=code
Should I refactor all those?